railties/lib/rails/application.rb

```diff
@@ -511,6 +515,11 @@ def validate_secret_key_config! #:nodoc:
 
         if secrets.secret_token.blank?
           raise "Missing `secret_key_base` for '#{Rails.env}' environment, set this value in `config/secrets.yml`"
+        else
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
+           secrets.secret_token is deprecated and will be removed in Rails 6.0.
+           Please use secrets.secret_key_base instead.
```

Comment:
Let's say: "Instead, assign a `secret_key_base` through either credentials, encrypted secrets or in `config/secrets.yml`."